test/helpers/file-helper.test.js

```diff
@@ -51,7 +51,7 @@ describe('--- FileHelper ---', () => {
     });
 
     it('returns the default content type for any other file', () => {
-      expect(FileHelper.getContentType('hello.json')).to.equal('application/octet-stream');
+      expect(FileHelper.getContentType('hello.json')).to.equal('application/json');
```

Comment:
We should probs change this file type to one that doesn't exist in the map so that the test makes sense 👍